worker/crates/action-sink/src/file/material.rs

```diff
@@ -0,0 +1,147 @@
+//! Material mangement
+
+use std::{hash::Hash, path::Path};
+
+use indexmap::IndexSet;
+use nusamai_gltf::nusamai_gltf_json;
+use nusamai_gltf::nusamai_gltf_json::{BufferView, MimeType};
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+#[derive(Debug, Serialize, Clone, PartialEq, Deserialize)]
+pub struct Material {
+    pub base_color: [f32; 4],
+    pub base_texture: Option<Texture>,
+    // NOTE: Adjust the hash implementation if you add more fields
+}
+
+impl Eq for Material {}
+
+impl Hash for Material {
```

Comment:
This is the type of
to be used. If the following code needs modification, please modify the following code.
https://github.com/reearth/reearth-flow/blob/main/worker/crates/types/src/geometry.rs#L265-L268

Reply:
Move to geometry.rs